documentation/src/main/asciidoc/topics/proc_encoding_protobuf.adoc

```diff
@@ -0,0 +1,24 @@
+[id='encoding-caches-protobuf_{context}']
+= Encoding caches as Protobuf
```

Comment:
I'd still say "as ProtoStream" instead of "as Protobuf".

Reply:
yeah that's more correct